CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
+- [#1326: Update tabs component to WCAG 2.1 compliant focus style](https://github.com/alphagov/govuk-frontend/pull/1326)
+- [#1361: Remove `govuk-focusable`, `govuk-focusable-fill` mixins, introduce `govuk-focus-text` mixin](https://github.com/alphagov/govuk-frontend/pull/1361)
 
-  ([PR #1443](https://github.com/alphagov/govuk-frontend/pull/1443))
 
-- Two unused images (`assets/images/icon-arrow-left.png` and
-  `assets/images/icon-important.png`) have been removed.
+### Update colours
 
-  ([PR #1473](https://github.com/alphagov/govuk-frontend/pull/1473))
+You can now use the following new colour variables that we've added to the [colour palette](https://design-system.service.gov.uk/styles/colour#main-colours):
 
-🆕 New features:
+- dark-blue
+- dark-grey
+- mid-grey
+- light-grey
 
-- IE 8 now falls back to Arial rather than Transport
+5 Sass colour variables no longer exist. Replace the following colour variables if you're using Sass:
 
-  All of the browsers we support use the WOFF or WOFF2 fonts, except for IE 8
-  which requires Embedded Open Type (.eot). The EOT version of the font is
-  considerably larger, which impacts performance for users who are likely to be
-  on older machines.
+Colour variable removed|Suggested replacement
+| --- | --- |
+bright-red|red
+grey-1|dark-grey
+grey-2|mid-grey
+grey-3|light-grey
+grey-4|light-grey
 
-  Given IE8 only represents a very small percentage of traffic we get, we now
-  fallback to Arial instead.
+You should check the [contrast ratio of your colours](https://design-system.service.gov.uk/styles/colour#colour-contrast).
 
-  Thanks to [@Nooshu](https://github.com/Nooshu) for their
-  [recommendation to drop EOT files for older IE](https://github.com/alphagov/govuk-frontend/pull/1356#issuecomment-494727525).
+If you're not using Sass, change the values of the following colours:
 
-  ([PR #1434](https://github.com/alphagov/govuk-frontend/pull/1434))
+Colour name|Old value|Replace with
+| --- | --- | --- |
+purple|#2e358b|#4c2c92
+red|#b10e1e|#d4351c
+yellow|#ffbf47|#ffdd00
+green|#006435|#00703c
+light-blue|#2b8cc4|#5694ca
+blue|#005ea5|#1d70b8
 
-- Checkboxes and radios use a new focus state with a thicker border. The
-  transparent outline, previously required to show the focus state when custom
-  colour schemes are used, has been removed.
+We've also changed the background of the following components:
 
-  ([PR #1316](https://github.com/alphagov/govuk-frontend/pull/1316))
+- buttons - `green` instead of a custom green
+- confirmation panels - `green` instead of `turquoise`
+- links in their hover state - `dark-blue` instead of `light-blue`
 
-- A new setting `$govuk-use-legacy-palette` has been added, which by default
-  will be true if any of the `$govuk-compatibility-*` settings are true.
+If you're using legacy projects like GOV.UK Elements, you can keep your current colours by [turning on compatibility mode](https://github.com/alphagov/govuk-frontend/docs/installation/compatibility.md).
```

Comment:
Would it be worth adding "while you migrate your service to use GOV.UK Frontend". We probably don't want to make it sound a viable option to continue using legacy codebases as they don't meet WCAG 2.1.

Reply:
I agree but I think we've covered this in the new compatibility page this links to, where it says "If you use these features, your service will not have GOV.UK Frontend's many accessibility and usability improvements."

Reply:
Ok no worries this is not blocking 👍